src/cloud/live/sens-data-over.md

```diff
@@ -60,6 +62,17 @@ All system configurations are set during build and deploy according to the follo
 1. If an environment variable does not exist, use the configuration from a `MAGENTO_CLOUD_RELATIONSHIPS` name-value pair in the [`.magento.app.yaml` file][app-yaml]. Ignore the default configuration.
```

Comment:
Why `MAGENTO_CLOUD_RELATIONSHIPS` is mentioned here? This variable contains only connection data to services not system configurations.

Reply:
See above.